### ISSUE TITLE
Regarding video recording preview.

### ISSUE THREAD

**Issue by shwetachauhan-simform** [infinum/Android-GoldenEye#47]
Can I remove left and right space while we record video? While record video I want preview in autofit. Is it possible?

**Comment by domagojkorman** [comment]
Hi @shwetachauhan-simform ,

You should be able to set AUTO_FIT or AUTO_FILL scaling depending on your needs.

https://github.com/infinum/Android-GoldenEye/blob/28d0416602432227d099f6bbf9e2d544401d59ca/goldeneye/src/main/java/co/infinum/goldeneye/config/SizeConfig.kt#L48

`previewScale` is accessed via `config` as all other properties.

**Comment by ShwetaChauhan18** [comment]
Yes, that’s working after video recording complete but while we are recording video screen width become small and there is black space in left and right?

Is it possible while recording is ongoing?

**Comment by shwetachauhan-simform** [comment]
ok, got it.
Thanks @domagojkorman
